terminal_connectiontoken.go

```diff
@@ -0,0 +1,12 @@
+package stripe
+
+// TerminalConnectionTokenParams is the set of parameters that can be used when creating a terminal connection token.
+type TerminalConnectionTokenParams struct {
+	Params             `form:"*"`
+	OperatorAccount    *string         `form:"operator_account"`
+}
+
+// TerminalConnectionToken is the resource representing a Stripe terminal connection token.
+type TerminalConnectionToken struct {
+	ConnectionToken    string          `json:"connection_token"`
```

Comment:
So a `ConnectionToken` comes back with a single field that's also called `ConnectionToken` and there's no `object` field?

This doesn't seem all that great from an API perspective because (1) activation token is both a resource and a field in a resource, and (2) it doesn't future proof things very well in case you want to add more fields to this resource in the future. Did you guys consider naming this `secret` after the field of that name in ephemeral keys?